### ISSUE TITLE
Accessibility: Need more visual focus indicators for keyboard-only users

### ISSUE THREAD

**Issue by terichadbourne** [ipfs-shipyard/ipfs-share-files#66]
When using the site with only a keyboard, there are not enough visual cues to the user that focus is changing to a new link, the add files button, etc. (I do see focus rings on the IPFS logo at top left and PL logo at bottom right, but those are the only ones I can spot.) Because Chrome displays the URL of links you are hovering over at the bottom left, I have some clues about where I am while tabbing IF it was link, but we shouldn't expect a keyboard-only user to rely on these, and that doesn't help for the Add File button. 

Whatever cue causes the links to turn a pointer into a hand on the links, make the Add File circle change color, etc. for a mouse user (presumably an on hover event) should be updated to apply also (or differently) when a keyboard user has focused on an element.

This is a problem on the main IPFS website as well (see https://github.com/ipfs/website/issues/284).

**Comment by jessicaschilling** [comment]
Closing in favor of the meta-accessibility issue https://github.com/ipfs/ipfs-gui/issues/28.
